.packit.yml

```diff
@@ -35,15 +35,14 @@ jobs:
       - fedora-all
     specfile_path: bpfman.spec
   - job: copr_build
-    trigger: release
-    branch: main
+    trigger: pull_request
```

Comment:
if not specified, this would trigger builds for pull request with any target branch.

Reply:
Right I just did that for some testing 
I'll remove that bit.